### ISSUE TITLE
Zomus instal from pip missing folder

### ISSUE THREAD

**Issue by jairot** [actmd/zoomus#1]
I just installed the client using pip and it doesn't runs, because the component folder is missing in the install.

```python
In [2]: from zoomus import ZoomClient
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-2-a9bd3832bb69> in <module>()
----> 1 from zoomus import ZoomClient

/home/jtrad/play/mejorpromedio/local/lib/python2.7/site-packages/zoomus/__init__.py in <module>()
      7 __version__ = '0.1.1'
      8 
----> 9 from zoomus.client import ZoomClient

/home/jtrad/play/mejorpromedio/local/lib/python2.7/site-packages/zoomus/client.py in <module>()
      4 __email__ = "prschmid@act.md"
      5 
----> 6 from zoomus import (
      7     components,
      8     util)

ImportError: cannot import name components
```

**Comment by prschmid** [comment]
Can you give me a little more details as to what you were doing? I did a clean pip install on a new virtualenv (we use [virtualenvwrapper](https://virtualenvwrapper.readthedocs.org/en/latest/)) and everything seems to be working correctly.

```sh
# Make new virtual environment
$ mkvirtualenv zoomus-test

# Install zoomus in the environment
$ pip install zoomus

# Show what is installed in the current environment
$ pip freeze
requests==2.9.1
wheel==0.24.0
zoomus==0.1.1

# Start python and test it
$ python
Python 2.7.10 (default, Oct 23 2015, 18:05:06) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from zoomus import ZoomClient
>>> client = ZoomClient('API_KEY', 'API_SECRET')
>>> rv = client.user.list()
>>> print rv.text
{"error":{"code":200,"message":"Invalid api key or secret"}}
```
(Note, the last error is expected since I didn't provided it with a real ``API_KEY`` and ``API_SECRET``

**Comment by prschmid** [comment]
I haven't heard anything back, so I'm closing this issue.

**Comment by prschmid** [comment]
@jairot Looks like the issue was in the building of the .egg file pushed to Pypi. This was fixed in 846e00b (release 0.1.2).
